viewer_server/client_db_access_handler.py

```diff
@@ -661,10 +666,13 @@ def getCheckerDoc(self, checkerId):
 
         text = "No documentation found for checker: " + checkerId + \
                "\n\nPlease refer to the documentation at the "
+        sa_link = "http://clang-analyzer.llvm.org/available_checks.html"
+        tidy_link = "http://clang.llvm.org/extra/clang-tidy/checks/list.html"
+
         if "." in checkerId:
-            text += "[ClangSA](http://clang-analyzer.llvm.org/available_checks.html)"
+            text += "[ClangSA]("+sa_checkers_link+")"
```

Comment:
Here and on the next link, it might be better to have space around the `operator+`.